lib/src/main/kotlin/at/bitfire/ical4android/AndroidEvent.kt

```diff
@@ -320,6 +301,10 @@ abstract class AndroidEvent(
         event.location = row.getAsString(Events.EVENT_LOCATION)
         event.description = row.getAsString(Events.DESCRIPTION)
 
+        row.getAsString(Events.UID_2445)?.let { uid ->
+            event.uid = uid
+        }
```

Comment:
I think this duplicates the `event.uid = row.getAsString(Events.UID_2445)` a few lines above.